packages/expo-ui/android/src/main/java/expo/modules/ui/ShapeView.kt

```diff
@@ -0,0 +1,158 @@
+package expo.modules.ui
+
+import android.content.Context
+import expo.modules.kotlin.views.ExpoComposeView
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableFloatStateOf
+import androidx.compose.ui.graphics.Color
+import expo.modules.kotlin.AppContext
+import expo.modules.kotlin.views.ComposeProps
+import androidx.compose.foundation.layout.Box
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawWithCache
+import androidx.graphics.shapes.RoundedPolygon
+import androidx.graphics.shapes.toPath
+import androidx.compose.ui.graphics.asComposePath
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.graphics.Path
+import androidx.graphics.shapes.CornerRounding
+import androidx.graphics.shapes.circle
+import androidx.graphics.shapes.pill
+import androidx.graphics.shapes.pillStar
+import androidx.graphics.shapes.rectangle
+import androidx.graphics.shapes.star
+import expo.modules.kotlin.types.Enumerable
+import android.graphics.Color as GraphicsColor
+
+enum class ShapeType : Enumerable {
+  STAR,
+  PILL_STAR,
+  PILL,
+  CIRCLE,
+  RECTANGLE,
+  POLYGON
+}
```

Comment:
Something to consider to be more in-line with our other modules. We usually use lowercase letters on the JS side. In here it doesn't make that much difference though, since users won't be using the enum directly so I'm fine with going either way.

```suggestion
enum class ShapeType(val value: String) : Enumerable {
  STAR("star"),
  PILL_STAR("pillStar"),
  PILL("pill"),
  CIRCLE("circle"),
  RECTANGLE("rectangle"),
  POLYGON("polygon")
}
```